testdata/p4_14_errors_outputs/nasty_meta.p4-stderr

```diff
@@ -0,0 +1,4 @@
+nasty_meta.p4(6):syntax error, unexpected {, expecting (
```

Comment:
Something is wrong, this is not the error message that you should expect.

Reply:
thanks for catching this. updated to the proper error message.